_data/projects/Smack.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: Smack
+desc: >-
+  Smack is an open source, highly modular, easy to use, XMPP client library written in Java for Java SE compatible JVMs and Android.
+site: https://www.igniterealtime.org/projects/smack/
+tags:
+- xmpp
+- java
+- im
+- client
+upforgrabs:
+  name: Teaser Tasks
+  link: https://igniterealtime.atlassian.net/issues/?filter=10003
```

Comment:
Not a blocker on merging, but when I follow this link I get this error:

`The requested filter doesn't exist or is private. Try logging in.`

Does that sound right?

Reply:
The JIRA filter should be public now.

Reply:
Still no change on my end 🤔 

<img width="1223" src="https://user-images.githubusercontent.com/359239/104186457-cd7f6e80-53ec-11eb-991d-5c2feb4d53a3.png">

Reply:
Now it should work. Sorry.